File: webservice.py
from flask import flask, request

app = flask(__name__)

@app.route ('/email', methods=['POST'])

def email ():
    
    print('To: ', request.form['to'])
    print('From: ', request.form['from'])
    print('Date:', request.form['date'])
    print('Subject: ', request.form['subject'])
    print('Message-ID:', request.form['message-id'])
    return ''

File: templates/Webservice.php
<?php
  $to = $_POST['to'];
  $from = $_POST['from'];
  $date = $_POST['date'];
  $subject = $_POST['subject'];
  $message_id = $_POST['message_id'];
  $message = $_POST["message"];
 
  $body = "To view this email with images, click below. \n\n $message";
  $headers = "From: $from";

  mail ($to, $from, $date, $subject, $message_id, $body, $headers,);
  
  echo "Message Sent! <a href='index.html'>Click here</a> to send another email";
?>
